app/uk/gov/hmrc/helptosavefrontend/connectors/HelpToSaveConnector.scala

```diff
@@ -57,34 +61,58 @@ class HelpToSaveConnectorImpl @Inject() (http: WSHttp)(implicit ec: ExecutionCon
   val base64Encoder: Base64.Encoder = Base64.getEncoder
 
   def getEligibility()(implicit hc: HeaderCarrier): EitherT[Future, String, EligibilityCheckResult] =
-    handle(
+    handleGet(
       eligibilityURL,
       _.parseJson[EligibilityCheckResponse].flatMap(toEligibilityCheckResponse),
       "check eligibility",
       identity
     )
 
   def getUserEnrolmentStatus()(implicit hc: HeaderCarrier): Result[EnrolmentStatus] =
-    handle(enrolmentStatusURL, _.parseJson[EnrolmentStatus], "get user enrolment status", identity)
+    handleGet(enrolmentStatusURL, _.parseJson[EnrolmentStatus], "get user enrolment status", identity)
 
   def enrolUser()(implicit hc: HeaderCarrier): Result[Unit] =
-    handle(enrolUserURL, _ ⇒ Right(()), "enrol users", identity)
+    handleGet(enrolUserURL, _ ⇒ Right(()), "enrol users", identity)
 
   def setITMPFlag()(implicit hc: HeaderCarrier): Result[Unit] =
-    handle(setITMPFlagURL, _ ⇒ Right(()), "set ITMP flag", identity)
+    handleGet(setITMPFlagURL, _ ⇒ Right(()), "set ITMP flag", identity)
 
   def storeEmail(email: Email)(implicit hc: HeaderCarrier): Result[Unit] = {
     val encodedEmail = new String(base64Encoder.encode(email.getBytes()))
-    handle(storeEmailURL(encodedEmail), _ ⇒ Right(()), "store email", identity)
+    handleGet(storeEmailURL(encodedEmail), _ ⇒ Right(()), "store email", identity)
   }
 
   def getEmail()(implicit hc: HeaderCarrier): Result[Option[String]] =
-    handle(getEmailURL, _.parseJson[GetEmailResponse].map(_.email), "get email", identity)
+    handleGet(getEmailURL, _.parseJson[GetEmailResponse].map(_.email), "get email", identity)
+
+  def isAccountCreationAllowed()(implicit hc: HeaderCarrier): Result[Boolean] = {
+    handleGet(accountCreateAllowedURL, _.parseJson[Boolean], "account creation allowed", identity)
+  }
+
+  def updateUserCount()(implicit hc: HeaderCarrier): Result[Unit] = {
+    handlePost(updateUserCountURL, "", _ ⇒ Right(()), "update user count", identity)
+  }
+
+  private def handlePost[A, B](url:         String,
+                               body:        String,
+                               ifHTTP200:   HttpResponse ⇒ Either[B, A],
+                               description: ⇒ String,
+                               toError:     String ⇒ B
+  )(implicit hc: HeaderCarrier): EitherT[Future, B, A] =
+    EitherT(http.post(url, body).map { response ⇒
+      if (response.status == 200) {
+        ifHTTP200(response)
+      } else {
+        Left(toError(s"Call to $description came back with status ${response.status}. Body was ${response.body}"))
+      }
+    }.recover {
+      case NonFatal(t) ⇒ Left(toError(s"Call to $description failed: ${t.getMessage}"))
+    })
 
-  private def handle[A, B](url:         String,
-                           ifHTTP200:   HttpResponse ⇒ Either[B, A],
-                           description: ⇒ String,
-                           toError:     String ⇒ B
+  private def handleGet[A, B](url:         String,
```

Comment:
a lot of the logic in `handleGet` and `handlePost` can be factored out into a common method

Reply:
fixed